ocs_ci/templates/ocs-deployment/local-volume.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   labels:
     app: ocs-storagecluster
 spec:
+  tolerations:
+    - key: "node.ocs.openshift.io/storage"
```

Comment:
Was this changed only for 4.5? Or it's also for 4.4?

Reply:
added for 4.4 docs as well . https://gist.github.com/mmgaggle/09fe2c51689c4dda7598042cc25ae526#file-local-volume-yaml

https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.4/html-single/deploying_openshift_container_storage/index?lb_target=preview#creating-openshift-container-storage-cluster-on-amazon-ec2_aws-vmware